### DISCUSSION TITLE
config

### DISCUSSION THREAD

**Discussion by MXB2001** [web-scrobbler/web-scrobbler#2705] [Q&A] [answered]
How to reconfigure addon after initial run when it allows you to change some settings? Clicking/right clicking the addon icon in firefox doesn't have any option for that. Under the addons menu it also does not have a preferences section, only permissions and another one.

**Comment by MXB2001** [comment, marked as answer]
Just found it (at last). Go to manage in the right click on icon menu and from there find preferences. Strange, I could swear I did not see it before when I searched...

**Reply by isrl03** [reply]
When I right click on it I can't find a preferences option at all...

**Reply by isrl03** [reply]
> <img alt="image" width="189" src="https://user-images.githubusercontent.com/91444088/134836952-2df74d0c-8c4e-4621-bb8c-7b8608fabc60.png">

This is all I see

**Reply by Earthbadgerr** [reply]
> > <img alt="image" width="189" src="https://user-images.githubusercontent.com/91444088/134836952-2df74d0c-8c4e-4621-bb8c-7b8608fabc60.png">
> 
> This is all I see

Better later than never. Right-click the icon and select manage extension. At the top next to the enable slider theres 3 dots. (see image) Click Option and it should open into the options.
![image](https://user-images.githubusercontent.com/112113419/186736635-33604db8-99ab-4f48-ae7a-98b9ff3fc9b4.png)
